Sources/WebUI/RefreshControl.swift

```diff
@@ -0,0 +1,26 @@
+import WebKit
+
+/// This is a workaround for the absence of refresh control in macOS.
+#if canImport(UIKit)
+typealias RefreshControl = UIRefreshControl
+#else
+struct RefreshControl {
+    enum ControlEvent {
+        case valueChanged
+    }
+
+    func addTarget(_ target: Any?, action: Selector, for controlEvents: ControlEvent) {}
+    func endRefreshing() {}
+}
+extension WKWebView {
```

Comment:
nits: It might be better to have blank lines.

```suggestion
}

extension WKWebView {
```

Reply:
Fixed in [c2a2d29](https://github.com/cybozu/WebUI/pull/54/commits/c2a2d29f102cbb0e03d13b6b3181bc3b00bb1744)